tests/request_matching.rs

```diff
@@ -0,0 +1,74 @@
+use wiremock::matchers::{header, headers, method};
```

Comment:
I generally prefer to have `one test = one assertion` - could you split them up?